project/lib/screens/sign_in_screen.dart

```diff
@@ -12,11 +13,13 @@ import '../data/example_data.dart';
 
 /// Screen/Scaffold for signing in and signing up .
 class SignInScreen extends StatelessWidget {
+  SignInScreen({super.key});
+
   /// Named route for this screen.
-  static const routeName = "/";
+  static const routeName = "/signin";
 
   /// Creates an instance of [SignInScreen].
```

Comment:
Move documentation to the constructor :)

Reply:
Is fixed in latest commit 👍